docs/identities/get-started/sign-in.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: sign-in
+title: Implementing user login
+sidebar_label: Sign in
+---
+
+# Sign in
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { getSdkUrl } from "@site/src/hooks"
+import CodeBlock from "@theme/CodeBlock"
+import { FrameworkCodeTabs } from '@site/src/components/GuidesComponents'
+import jsLogin from '!!raw-loader!./_common/code-examples/js/login.js'
+import goLogin from '!!raw-loader!./_common/code-examples/go/login_handler.go'
+import nextjsLogin from '!!raw-loader!./_common/code-examples/nextjs/login.tsx'
+```
+
+The sign in flow follows the same pattern as the [sign up flow](/docs/identities/get-started/sign-up) but instead of redirecting
+to the registration page, it redirects to the login page.
+
+```mdx-code-block
+<FrameworkCodeTabs>
+<TabItem value="javascript">
+```
+
+```mdx-code-block
+<CodeBlock language="js"> {jsLogin} </CodeBlock>
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="nextjs">
+```
+
+```tsx
+export default function Page() {
+  const [session, setSession] = useState<Session | null>(null)
```

Comment:
I am following the guide "step by step" and it  does not explain on which port my app should run, which modules beside ory client I should use and the config code example is broken.

Reply:
Added a "What you'll need" section to set some expectation for the reader